src/App.tsx

```diff
@@ -1,9 +1,38 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+interface Movie {
+  title: string;
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string,
+}
+
+export function filterMovies(movies: Movie[], query: string): Movie[] {
+  const visibleMovies = movies.filter(movie => {
+    const prepareTitle = movie.title.toLowerCase();
+    const prepareDescription = movie.description.toLowerCase();
+    const prepareQuery = query.toLowerCase().trim();
+
+    return prepareTitle.includes(prepareQuery)
+      || prepareDescription.includes(prepareQuery);
```

Comment:
might be, but I would create function like 
`const checkString = (string) => string.toLowerCase().includes(query.toLowerCase().trim())`
and use it for title and description